docs/installing.md

```
@@ -0,0 +1,186 @@
+TO:
+
+``` shell
+defaults.ctl.card 1
+defaults.pcm.card 1
+```
+
+**2.** Install LedFx and all the dependencies using our [LedFx Bash
+Install Script](https://install.ledfx.app):
+
+``` console
+$ curl -sSL https://install.ledfx.app/ | bash
```

Markdownlint (0.35.0)

Dollar signs used before commands without showing output

(MD014, commands-show-output)